src/query/procedure/fmt.hpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#pragma once
+
+#include <fmt/ostream.h>
+
+#include "mg_procedure.h"
+
+template <>
+class fmt::formatter<mgp_log_level> {
```

Comment:
Is this duplicated from an operator<< or is this a completely new stringify?
Note from Gareth, we can create a ToString and use it in both palces